lib/spack/spack/test/svn_fetch.py

```diff
@@ -56,13 +63,18 @@ def test_fetch(
     t = mock_svn_repository.checks[type_of_test]
     h = mock_svn_repository.hash
     # Construct the package under test
-    spec = Spec('hg-test')
+    spec = Spec('svn-test')
     spec.concretize()
     pkg = spack.repo.get(spec, new=True)
-    pkg.versions[ver('hg')] = t.args
+    pkg.versions[ver('svn')] = t.args
```

Comment:
@alalazo I wonder if this is related to the transient `hg` fetch errors we've been seeing in Travis. I could see why it would be a problem if `hg_fetch` and `svn_fetch` both run in parallel at the same time.